src/features/create-drop/components/WalletComponent.tsx

```diff
@@ -1,29 +1,24 @@
 import { type CheckboxItem } from '@/components/Checkboxes';
+import { CoinIcon } from '@/components/CoinIcon';
```

Comment:
what is `CoinIcon`? Is this for the wallets?

Reply:
If it is, naming is unclear - we should fold the wallet icons in the wallet global section

Reply:
will refactor this